spec/features/question_form_spec.rb

```diff
@@ -51,4 +51,22 @@
       end
     end
   end
+
+  scenario "audio upload works", js: true do
+    visit new_question_path(locale: "en", mode: "m", mission_name: mission.compact_name)
+    fill_in "Code", with: "AQuestion"
+    fill_in "Title", with: "Jay's"
+    select "Text", from: "Type"
+
+    attach_file("Audio Prompt", File.absolute_path("spec/fixtures/media/audio/powerup.mp3"))
```

Comment:
Using `absolute_path` is an interesting technique, but it depends on the specs being run from the correct working directory. In general it's less brittle to use `Rails.root` or better `Rails.root.join` to construct paths like this.

But _even better_ would be a function for retrieving fixture files, which we have I think. See if you can find it.